.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+python:
+  - '3.6'
+
+install:
+  - pip install --upgrade sphinx
+
+script:
+  - ./scripts/update_docs.sh
+  - touch docs/.nojekyll
+
+branches:
+  only:
+    - master
+
+deploy:
+  provider: pages
```

Comment:
Can you add a link to https://docs.travis-ci.com/user/deployment/pages/ ?  I had to Google a bit to figure out what this was doing.

Reply:
Sorry for not including this link with the PR description. Added it now.